language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/codeaction/AbstractCodeActionTest.java

```diff
@@ -479,4 +483,31 @@ private boolean validateExtractCmd(JsonObject actualCommand, JsonArray actualArg
         actualCommand.add("arguments", newArgs);
         return false;
     }
+
+    private static final PrintStream printStream = System.out;
+
+    private boolean validateCreateConfigTomlCommand(JsonObject actualCommand, JsonArray actualArgs, JsonArray expArgs,
+                                                                                      Path sourceRoot) {
+        String actualFilePath = actualArgs.get(0).getAsString().replace(sourceRoot.toString(), "");
```

Comment:
Don't we need to assert that we have `actualArgs` > 0?

Reply:
We check the arg size in the line 442.